### DISCUSSION TITLE
MPACT Q&A

### DISCUSSION THREAD

**Discussion by robertsamples** [BalunasLab/mpact#2] [Announcements]
Submit questions in this discussion board.

**Comment by Judez10** [comment]
Hi Robert,
I've followed the manual on generating the files needed but a KeyError message keeps appearing when I try to run the analysis. I've included screenshots of the files and the error message.

Peak list (generated with MZmine3):
![peak list](https://github.com/BalunasLab/mpact/assets/147752708/690651b8-11eb-435c-b85a-7bb2f1cd9507)
Sample list:
![sample list](https://github.com/BalunasLab/mpact/assets/147752708/0d525ca7-b013-4673-b998-7112afdf0f1d)
Metadata file:
![metadata](https://github.com/BalunasLab/mpact/assets/147752708/5a8a3403-f134-4e23-8b10-c32a97986e5d)
Error message:
![keyerror](https://github.com/BalunasLab/mpact/assets/147752708/9e36a467-fb62-4621-97a0-deff3627a2f5)
